src/main.ts

```diff
@@ -5,6 +5,10 @@ import helmet from "helmet";
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors({
+    origin : ['*', 'http://localhost:4200'],
```

Comment:
Need to convert this to an environment variable.

Reply:
done